linux/LinuxProcessList.c

```diff
@@ -479,54 +479,33 @@ static bool LinuxProcessList_readSmapsFile(LinuxProcess* process, const char* di
    //http://elixir.free-electrons.com/linux/v4.10/source/fs/proc/task_mmu.c#L719
    //kernel will return data in chunks of size PAGE_SIZE or less.
 
-   char buffer[CRT_pageSize];// 4k
-   ssize_t nread=0;
+   char buffer[128];
```

Comment:
128 might not be enough
E.g. `cat /proc/$$/smaps` contains (length 120)
```
7fea52867000-7fea5286a000 r--p 00000000 fe:01 929866                     /usr/lib/x86_64-linux-gnu/libnss_files-2.31.so
```

And too long lines might break the `fgets()` logic

Reply:
Doubled buffer size and copied line skip logic from my OpenVZ status file parser.
Long files would only have broken things if you had one of the keywords exactly after a buffer boundary.